integration-tests/basic-test-app/app/(tabs)/index.tsx

```diff
@@ -55,48 +95,103 @@ const HomeScreen = () => {
     });
   }, []);
 
+  const handleAddSimpleBreadcrumb = () => {
+    addBreadcrumb("A SIMPLE BREADCRUMB");
+  };
+  const handleAddComplexBreadcrumb = () => {
```

Comment:
what's different about this breadcrumb test case?

Reply:
There is no difference, but with other name. 
I didnt want to name it Breadcrumb 1 and 2.

I used it to test that two different breadcrumb be recorded